lib/templates/alarms.rb

```diff
@@ -160,18 +161,18 @@
       Condition "Condition#{alarmHash}" if conditions.length > 0
       Type('AWS::CloudWatch::Alarm')
       Property('ActionsEnabled', FnIf('MonitoringDisabled', false, FnFindInMap("#{alarmHash}",'ActionsEnabled',Ref('EnvironmentType'))))
-      Property('AlarmActions', alarmActions)
+      Property('AlarmActions', alarmActions) unless alarmActions.nil?
```

Comment:
I don't think `alarmActions` will ever be nil as it will always to `crit` sns endpoint if nothing is set
https://github.com/base2Services/cfn-monitor/blob/fc2caaeee47ac5207d49e9e5eec4b4c602c77342/lib/templates/alarms.rb#L84

Reply:
Ah yes I see. My thinking was since `params['AlarmActions']` would be set to the `none` key it'd pass that, but none is nil anyway. Do you have a suggestion for this? I can't really think of many cases where you would want the none action on an Alarm state anyway.